sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlArrayTest.java

```diff
@@ -80,11 +81,11 @@ public void testSelectArrayValue() {
   public void testProjectArrayField() {
     PCollection<Row> input = pCollectionOf2Elements();
 
-    RowType resultType =
+    Schema resultType =
         RowSqlType
             .builder()
             .withIntegerField("f_int")
-            .withArrayField("f_stringArr", SqlTypeCoders.VARCHAR)
+            .withArrayField("f_stringArr", SqlTypeName.VARCHAR)
```

Comment:
<!--thread_id:cc_178347126_t; commit:79c95678e593da730ba0472b77304ec1f916245e; resolved:0-->
<!--section:context-quote-->
> **akedin** wrote:
> I think that we should not use Calcite types in the public APIs. Especially when the schema is now in core.

<!--section:body-->
This is calling into the RowSqlType builder, who's job it is to convert using CalciteUtils. Alternatively we could eliminate that builder and have all callers just call the core builder using CalciteUtils - does that sound better?

Reply:
I'll remove RowSqlTypeBuilder in a follow-in PR